Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ButtonData.xml

```diff
@@ -5515,7 +5515,7 @@
         <Icon value="Assets\Textures\btn-ability-terran-emergencythrusters.dds"/>
         <AlertIcon value="Assets\Textures\btn-ability-terran-emergencythrusters.dds"/>
         <EditorCategories value="Race:Terran"/>
-        <TintRacially value="1"/>
+        <TintRacially value="0"/>

```

Comment:
0 is the default option for this flag, you can just remove the line (same for the change above).